scripts/variantstore/gvs-overview.md

```diff
@@ -0,0 +1,165 @@
+# Genomic Variant Store pipeline overview
+ 
+| Pipeline Version | Date Updated | Documentation Authors | Questions or Feedback |
+| :----: | :---: | :----: | :--------------: |
+| [GvsJointVariantCalling](https://github.com/broadinstitute/gatk/blob/rc-vs-483-beta-user-wdl/scripts/variantstore/wdl/GvsJointVariantCalling.wdl) | June, 2022 | [Kaylee Mathews](mailto:kmathews@broadinstitute.org) and [Aurora Cremer](mailto:aurora@broadinstitute.org) | If you have questions or feedback, contact the [Broad Variants team](mailto:variants@broadinstitute.org) |
+ 
+![Diagram depicting the Broad Genomic Variant Store workflow. Sample GVCF files are imported into the core data model. A filtering model is trained using Variant Quality Score Recalibration, or VQSR, and then used to extract cohorts and produce sharded joint VCF files. Each step integrates BigQuery and GATK tools.](/scripts/variantstore/genomic-variant-store_diagram.png)
+
+
+## Introduction to the Genomic Variant Store pipeline
+
+The [Genomic Variant Store (GVS)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/gvs-product-sheet.pdf) was developed by the Data Sciences Platform at the Broad Institute of MIT and Harvard as a solution for variant discovery at scales previously inaccessible to most researchers due to large cohort sizes and high computational and financial costs. The Broad GVS is powered by BigQuery and currently supports the creation of joint callsets of up to 100,000 human genomes faster, cheaper, and more reliably than existing solutions.
```

Comment:
```suggestion
The [Genomic Variant Store (GVS)](https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/gvs-product-sheet.pdf) was developed by the Data Sciences Platform at the Broad Institute of MIT and Harvard as a solution for variant discovery at scales previously inaccessible to most researchers due to large cohort sizes and high computational and financial costs. The Broad GVS is powered by BigQuery and creates joint callsets of up to 100,000 human genomes faster, more cheaply, and more reliably than existing solutions.
```

Reply:
I think you want cheaply here to match the verb

Reply:
The first sentence seems complex and wordy to me, maybe it's the use of "inaccessible".   You're the expert here, so whatever you think is best.  But maybe remove the attribution and simplify the motivation?  

> we developed to address scale, cost and operational challenges in joint calling in large whole genome joint callsets.

Reply:
Also 

> than existing solutions

If we're going to say this, we might need to show our work (e.g. compare to GLnexus, sentieon, etc).  Maybe it would be better to say "previous solutions" or something?

Reply:
Great suggestions. I made the suggested changes and simplified the first sentence.